core/src/main/python/wlsdeploy/tool/util/targets/additional_output_helper.py

```diff
@@ -88,7 +87,7 @@ def _create_file(template_name, template_hash, model_context, output_dir, except
     target_key = model_context.get_target()
     template_subdir = "targets/" + target_key + "/" + template_name
     template_path = path_utils.find_config_path(template_subdir)
-    output_file = File(output_dir, template_name)
+    output_file = File(output_dir, target_key + '-' + template_name)
```

Comment:
But there is also a k8s target, does it also generate domain.yaml ?   This also covers the vz-application.yaml

Reply:
The k8s target doesn't generate a resource yaml. I think the template / file name should be controlled by target.json, without the code making any adjustments. This makes it easy to change a target independently of the others.
If we've agreed to change vz also, that can be handled by renaming its file and target.json reference.

Reply:
Opinions from other reviewers?

Reply:
From the UI perspective, we need a predictable file name.  While the idea is interesting to be able to control the file name from the configuration file, it exposes the UI to potential errors if the configuration file name is not in sync with the UI code looking for the file (i.e., the UI code does not parse the WDT configuration file to determine the file names to look for).  What happens if the file name is not specified in the configuration file?  It seems like the code should gracefully handle that use case.  If so, then the filename handling code in question needs to be enhanced (rather than simplified).

Reply:
The file name is already specified in the configuration file and read from there in all cases. This change prepends the target name to the specified name, removing the ability to provide specific file name(s) for a target.

Reply:
OK.  Then the default names in the configuration files need to change.

From an API/SPI user perspective, what happens if the 3rd party overriding the default configuration file happens to omit the filename parameter?  Does the tool just fail?

Reply:
The `additional_output` field has the names of files that to convert from templates in target configuration dir to files in the output dir. If the user leaves this blank, no files are processed. Target -vz had multiple files at one point, driving some of these changes.

Reply:
ok, I can change the file name instead.